src/frames.c

```diff
@@ -950,6 +950,7 @@ int novas_app_to_geom(const novas_frame *frame, enum novas_reference_system sys,
       break;
     default:
       // nothing to do.
+      ;
```

Comment:
I can confirm that `-std=c99` does warn on the empty closing label. So you are correct that the `;` is necessary for some C versions...